autograder_django_backend/autograder/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+
+class Course(models.Model):
+    course_id = models.CharField(max_length=10)
+    name = models.CharField(blank=True, max_length=50)
+    description = models.TextField(blank=True)
+    section = models.CharField(max_length=5)
+    professor = models.ForeignKey(
+        "autograder.Professor", on_delete=models.CASCADE, related_name="courses"
+    )
+
+    def get_students_from_course(self):
+        return list(Student.objects.filter(courses=self))
+
+
+class Assignment(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    due_date = models.DateTimeField()
+
+    assignment_spec = models.TextField()
+    # uploaded professor test cases as a file
+
+    course = models.ForeignKey(
+        "Course", on_delete=models.CASCADE, related_name="assignments"
```

Comment:
Why does line 10 have to be autograder.Profesor, but line 26 is just course? Is this because Course is declared above assignment?